### PR TITLE
[RAPTOR-4944] Update Drum for python parameter input support, add R support

### PR DESCRIPTION
**Pull request by jmbannon**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Updates DRUM to the next minor version to support parameter files.
- Updated drop-in envs to look for parameter file in the input directory
- Added `test_fit_sh` test-case for parameter files
- Add parameter support for R, added test

### CODE REVIEW

File: tests/testdata/sklearn_transform_parameters.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "numeric_imputer_strategy": "constant",
   "numeric_imputer_constant_fill": -50.0,
   "numeric_standardize_with_mean": 0,
-  "categorical_fill": "unicode compatibility: 🚀"
+  "categorical_fill": "unicode compatibility: \uD83D\uDE80🚀"
```
Comment (4dahalibut) [inline tests/testdata/sklearn_transform_parameters.json:5]: what's this change for?

Comment (jmbannon) [reply tests/testdata/sklearn_transform_parameters.json:5]: weird, the unicode didn't render for me locally. I'll revert this file

**Review by 4dahalibut — APPROVED** [review]

**Comment by jmbannon** [comment]
jarvis test failed

**Comment by jmbannon** [comment]
jarvis test functional_inference_model_templates

**Review by yakov-g — APPROVED** [review]

**Pull request merged**